packages/xo-web/src/common/intl/messages.js

```diff
@@ -2213,6 +2216,14 @@ const messages = {
   recipeSshKeyLabel: 'SSH key',
   recipeNetworkCidr: 'Network CIDR',
 
+  // Audit
+  auditActionEvent: 'Action/Event',
+  auditDeletedUser: 'deleted ({ name })',
```

Comment:
If it's not specifically related to the Audit feature, don't namespace it so that it can be as generic/reusable as possible.